pkg/extensionController/controller_i_test.go

```diff
@@ -160,6 +160,15 @@ func (suite *ControllerITestSuite) TestUpgradeSucceeds() {
 	suite.Equal(&extensionAPI.JsUpgradeResult{PreviousVersion: "0.1.0", NewVersion: "0.2.0"}, result)
 }
 
+func (suite *ControllerITestSuite) TestUpgradeGettingScriptReturnsNil() {
+	suite.createExtensionBuilder().
+		WithUpgradeFunc("const script = context.metadata.getScriptByName('script'); return {previousVersion:'0.1.0',newVersion:'script is null = '+(script===null)};").
+		Build().WriteToFile(path.Join(suite.tempExtensionRepo, EXTENSION_ID))
+	result, err := suite.createController().UpgradeExtension(mockContext(), suite.exasol.GetConnection(), EXTENSION_ID)
+	suite.NoError(err)
+	suite.Equal(&extensionAPI.JsUpgradeResult{PreviousVersion: "0.1.0", NewVersion: "script is null = true"}, result)
+}
```

Comment:
Sorry, my suggestion is probably not working, but I hope it expresses my proposal ? 

```suggestion
func (suite *ControllerITestSuite) TestUpgradeGettingScriptReturnsNil() {
    String UNINSTALLED_SCRIPT = script;
    script_not_installed = "(script===null)";
	suite.createExtensionBuilder().
	    WithUpgradeFunc("const installation = context.metadata.getScriptByName('" //
			    +UNINSTALLED_SCRIPT //
			    'result = installation === null ? "null" : "other"; '
			    + "'); return {previousVersion:'0.1.0',newVersion:'result};").
		Build().WriteToFile(path.Join(suite.tempExtensionRepo, EXTENSION_ID))
	result, err := suite.createController().UpgradeExtension(mockContext(), suite.exasol.GetConnection(), EXTENSION_ID)
	suite.NoError(err)
	suite.Equal(&extensionAPI.JsUpgradeResult{PreviousVersion: "0.1.0", NewVersion: "script is null = true"}, result)
}
```